supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -21,14 +21,14 @@ Do not rely solely on a shell prompt in a sample command to indicate the require
 If you include a shell prompt to indicate that a user with root privileges must run the command, also include a statement about this requirement in the step text, the introductory text, or the prerequisites.
 ====
 +
-* When a sample command includes `sudo`, use the `$` prompt, not `#`, as shown in the following example: 
+* When a sample command includes `sudo`, use the `$` prompt, not `#`, as shown in the following example:
```

Comment:
```suggestion
* When a sample command includes `sudo`, use the dollar sign (`$`) to denote a regular user. Do not use the number sign (`#`) because this sign is reserved for the root user.
+
.Example command
[source,terminal]
----
```

Reply:
This is out of scope of my PR.

Reply:
NP. Thanks!